frontends/web/src/routes/account/add/add.tsx

```diff
@@ -122,6 +128,13 @@ export const AddAccount = () => {
   const renderContent = () => {
     switch (step) {
     case 'select-coin':
+      if (supportedCoins.length === 0) {
+        return (
+          <div className="text-center">
+            {t('connectKeystore.promptNoName')}
+          </div>
```

Comment:
```diff
diff --git a/frontends/web/src/routes/account/add/add.tsx b/frontends/web/src/routes/account/add/add.tsx
index a97d87496..62a4735b2 100644
--- a/frontends/web/src/routes/account/add/add.tsx
+++ b/frontends/web/src/routes/account/add/add.tsx
@@ -130,9 +130,9 @@ export const AddAccount = () => {
     case 'select-coin':
       if (supportedCoins.length === 0) {
         return (
-          <div className="text-center">
+          <Message type="info">
             {t('connectKeystore.promptNoName')}
-          </div>
+          </Message>
         );
       }
       return (
```

So it looks a bit more prominent:

![Screen Shot 2023-11-30 at 10 08 19](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/590d6c62-7c3c-43d0-8b99-efc0e2f1cc2e)